### DISCUSSION TITLE
Lost "add a new flashcard" option

### DISCUSSION THREAD

**Discussion by Zacerr** [fabd/kanji-koohii#228] [3. Bugs]
Hello :)
Unfortunately I have lost my button to "add a new flashcard". What could I do about this?
![problem](https://user-images.githubusercontent.com/96428219/146776970-8c6d6397-60be-4d7d-aaf5-6caa364b9046.png)

**Comment by fabd** [comment]
Yes thank you Zacerr

Sorry for the trouble :/

It appears to be a new bug. Coudln't see it after a quick glance at the code so I'll have to dig into it further later today.

In the meantime please go to [Manage Flashcards > Add Custom ](https://kanji.koohii.com/manage/addcustom) where you can entter something like `20 21 22` or `10-20` for example, to add range of cards at once.

**Reply by Zacerr** [reply]
Thank you for a quick reply! That’s what I’ll do :)
Also thank you for your work, we all appreciate it!

**Reply by fabd** [reply]
Thankfully this was a fairly simple bug. It should work now.

Thank you for the quick report!
